webserver/webserver/src/main/java/io/helidon/webserver/http/ServerResponse.java

```diff
@@ -190,6 +190,14 @@ default void send(Optional<?> entity) {
      */
     ServerResponseHeaders headers();
 
+    /**
+     * Response trailers (mutable).
+     * @return trailers
+     * @throws java.lang.IllegalStateException if client didn't ask for trailers with {@code TE: trailers} header in request
+     * or response doesn't contain trailer declaration headers {@code Trailer: <trailer-name>}
+     */
+    ServerResponseHeaders trailers();
```

Comment:
As I wrote above - I think this should be `Headers` and not `ServerResponseHeaders`

Reply:
Changed to ServerResponseTrailers direct extension of writeable headers